File: 백준/Silver/10431. 줄세우기/줄세우기.py
import sys
input=lambda:sys.stdin.readline().rstrip()
for _ in range(int(input())):
    arr=[*map(int,input().split())]
    total,l=0,len(arr)
    for i in range(1,l-1):
        for j in range(i+1,l):
            if arr[i]>arr[j]:
                arr[i],arr[j]=arr[j],arr[i]
                total+=1
    print(arr[0],total)

File: 백준/Silver/10431. 줄세우기/줄세우기.cc
#include <iostream>
#include <vector>
using namespace std;
int main(void) {
    int n, t, s, len, total;
    cin >> t;
    for (int i = 0; i < t; i++) {
        vector<int> arr(21);
        for (int j = 0; j < 21; j++)
            cin >> arr[j];
        len = arr.size(), total = 0;
        for (int j = 1; j < len - 1; j++) {
            for (int k = j + 1; k < len; k++) {
                if (arr[j] > arr[k]) {
                    s = arr[j];
                    arr[j] = arr[k];
                    arr[k] = s;
                    total += 1;
                }
            }
        }
        cout << arr[0] << ' ' << total << "\n";
    }
    return 0;
}